rotkehlchen/history/events/structures/base.py

```diff
@@ -190,11 +191,13 @@ def _serialize_base_tuple_for_db(self) -> tuple[str, str, HISTORY_EVENT_DB_TUPLE
             (
                 'history_events(entry_type, event_identifier, sequence_index,'
                 'timestamp, location, location_label, asset, amount, notes,'
-                'type, subtype, extra_data) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)'
+                'type, subtype, extra_data, ignored) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, '
+                "(CASE WHEN EXISTS (SELECT 1 FROM multisettings WHERE name = 'ignored_asset' AND value = ?) THEN 1 ELSE 0 END))"  # noqa: E501
             ), (
                 'UPDATE history_events SET entry_type=?, event_identifier=?, '
                 'sequence_index=?, timestamp=?, location=?, location_label=?, asset=?, '
-                'amount=?, notes=?, type=?, subtype=?, extra_data=?'
+                'amount=?, notes=?, type=?, subtype=?, extra_data=?, '
```

Comment:
same here and you are basically doing the multisettings subquery twice. Once here and once above.